### DISCUSSION TITLE
'Dex' alias for dexterity in @check like in @oldcheck

### DISCUSSION THREAD

**Discussion by somethingfinite** [Arx-Game/arxcode#376] [Ideas]
I do a lot of dex.

I miss my dex alias. I would love it if it came back so I could stop finding new ways to accidentally type dextertiy.

**Comment by Hierodule** [comment]
Getting every stat and skill to alias to the first three letters (or first three unique letters) would be an excellent quality of life improvement.

**Reply by somethingfinite** [reply]
Some definitely work better than others, given the naming conventions. But for a start, these seem like pretty easy wins.

Charm = CHA
Intellect = INT
Perception = PER
Stamina = STA
Strength = STR
Wits = WIT

Either way, shorthand helps lots.

**Reply by TehomCD** [reply]
Yeah, my intent with slowly converting traits to be part of the schema is so that I can have metadata associated with them, like aliases, prerequisites, costs, etc. It's something that's on the roadmap, though right now my focus is largely on backend changes to resolve some of our infrastructure problems, so things like that are a bit further out.
